generators/client/templates/angular/webpack/_webpack.dev.js

```diff
@@ -1,11 +1,41 @@
 const webpack = require('webpack');
+const path = require('path');
 const commonConfig = require('./webpack.common.js');
+const writeFilePlugin = require('write-file-webpack-plugin');
 const webpackMerge = require('webpack-merge');
 const BrowserSyncPlugin = require('browser-sync-webpack-plugin');
 const ExtractTextPlugin = require("extract-text-webpack-plugin");
 const ENV = 'dev';
+const execSync = require('child_process').execSync;
+const fs = require('fs');
+const ddlPath = './<%= BUILD_DIR %>www/vendor.json';
+
+if (!fs.existsSync(ddlPath)) {
+    execSync('webpack --config webpack/webpack.vendor.js');
+}
 
 module.exports = webpackMerge(commonConfig({env: ENV}), {
+    devtool: 'cheap-module-source-map',
```

Comment:
inline-source-map or cheap-module-eval-source-map might be better for dev as they would be faster to build. https://webpack.github.io/docs/configuration.html#devtool